devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -884,6 +886,21 @@ bool WholeBodyDynamicsDevice::loadSettingsFromConfig(os::Searchable& config)
     yarp::os::Property prop;
     prop.fromString(config.toString().c_str());
 
+    if (prop.check("devicePeriodInSeconds"))
+    {
+	if(!prop.find("devicePeriodInSeconds").isDouble())
```

Comment:
There is a strange indentation here, could it be space/tabs?

Reply:
https://github.com/robotology/whole-body-estimators/pull/130/commits/56f2583a0a89f00dcbd89126238c661c3bbacfc3